test/contrib/emscripten/test_emscripten.py

```diff
@@ -947,3 +948,26 @@ def count_calls(self, *args, **argv):  # type: ignore[no-untyped-def]
         assert count == 6
 
     pyodide_test(selenium_coverage, testserver_http.http_host, find_unused_port())
+
+
+@install_urllib3_wheel()
+def test_insecure_requests_warning(
+    selenium_coverage: typing.Any, testserver_http: PyodideServerInfo
+) -> None:
+    @run_in_pyodide  # type: ignore[misc]
+    def pyodide_test(selenium_coverage, host: str, port: int, https_port: int) -> None:  # type: ignore[no-untyped-def]
+        import urllib3
+        import urllib3.exceptions
+
+        http = urllib3.PoolManager()
+
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
CI fails because you haven't imported `warnings`

Reply:
Thanks! Fixed in https://github.com/urllib3/urllib3/pull/3333/commits/2758cab1668a86f51d873ea9d6041ad558c07a09